system/Router/Router.php

```diff
@@ -328,9 +328,7 @@ protected function checkRoutes(string $uri): bool
             return false;
         }
 
-        $uri = $uri === '/'
-            ? $uri
-            : ltrim($uri, '/ ');
+        $uri = trim($uri, '/ ') ?: '/';
```

Comment:
Minor typo I think why tests are failing.
```suggestion
        $uri = ltrim($uri, '/ ') ?: '/';
```

Reply:
@paulbalandan 
The following two code is not the same.
What if `$uri` is `"0"`.

```php
$uri = $uri === '/'
            ? $uri
            : trim($uri, '/ ');
```
```php
$uri = trim($uri, '/ ') ?: '/';
```

Reply:
@najdanovicivan This would make all tests pass.

```suggestion
        $uri = $uri === '/'
            ? $uri
            : trim($uri, '/ ');
```

Reply:
That was my initial implementation. But we thought one liner will do the job. I completely missed out the empty string or '0' string cases

@paulbalandan the whole idea with this is to change from ltrim to trim so that urls with trailing slash are properly routed